tests/e2e/vsan_test.go

```diff
@@ -77,12 +82,24 @@ func (s *VsanTestSuite) TearDownTest(c *C) {
 
 var _ = Suite(&VsanTestSuite{})
 
+func (s *VsanTestSuite) newCName() string {
+	cname := inputparams.GetUniqueContainerName("vsan_test")
+	s.containerList = append(s.containerList, cname)
+	return cname
+}
+
 // Steps:
 // 1. Create a valid vsan policy
 // 2. Volume creation with valid policy should pass
 // 3. Valid volume should be accessible
+// 4. Start a container to mount the volume, and write message to the volume
+// 5. Start another container to mount the volume, and read message from volume and verify the
```

Comment:
I'm confused here: I don't see the test logic related to this step - start *another* container to mount the volume. Can you please clarify?